### PR TITLE
Addition of github profile

### PR DESCRIPTION
**Pull request by rtucker-mozilla**
Per: https://bugzilla.mozilla.org/show_bug.cgi?id=941086

We could very much use a way to link users to their GitHub profiles. One way to do this would be to have them self-provision said information in Addressbook, using a new GitHub profile field.

Any method that permits machine-automated querying of LDAP users to find their GitHub account is sufficient.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -360,6 +360,18 @@ a:hover {
         left: 1px;
         content: "\f0c0";
     }
+    .vcard .githubProfile:before {
+        position:absolute;
```
Comment (tofumatt) [inline css/style.css]: Could you add a space after the colon here? Nitpicky, but you'll want to squash your commits anyway so might as well :smile:

**Comment by tofumatt** [comment]
Cool! r+wc

Can you squash this into one commit? Thanks ^_^

**Comment by rtucker-mozilla** [comment]
Looks like I couldn't get the rebase to work as requested.
If it's really important to only merge one commit feel free to fork my fork, rebase and then merge?

**Comment by rtucker-mozilla** [comment]
Created PR #17 squashing these commits

**Pull request closed without merging**